templates/default/flyingduck.hcl.erb

```diff
@@ -10,8 +10,8 @@ services = [
             id = "flyingduck-check"
             name = "Health check for flyingduck"
             http = "http://localhost:<%= node['flyingduck']['monitoring'] %>/metrics"
-            interval = "20s"
-            timeout = "20s"
+            interval = "180s"
```

Comment:
We shouldn't increase the timeouts, we should actually use the node FQDN instead of the service consul name to start the server since we have SAN work is merged already. So I guess you need to do the following:
1. Use `node["fqdn"]` instead of consul domain name
    https://github.com/logicalclocks/flyingduck-chef/blob/master/recipes/default.rb#L91
2.  Remove common_name, so it will be added by default with SAN work 
https://github.com/logicalclocks/flyingduck-chef/blob/master/recipes/default.rb#L97
https://github.com/logicalclocks/flyingduck-chef/blob/master/recipes/default.rb#L108

@kouzant any thing else i'm missing here. 
Once fixed on chef, I can fix it also on ec2-init

Reply:
No, @maismail you are correct. Remove `common_name` from the Chef resource. When the certificate is generated in Hopsworks CA it will have CommonName=FQDN - which is needed because certificates' Subject must be unique - and among other Subject Alternative Names `flyingduck.service.consul` So essentially the certificate will be valid for the FQDN ie `hopsworks0.logicalclocks.com` and the Consul domain name `flyingduck.service.consul` (cc @tdoehmen )

Reply:
I had a problem with master just now, where kafka on external clients are trying to connect to
kafka.consul.service

I expect i will have the same problem here with flyingduck.consul.service?